geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -85,7 +97,20 @@ case class CoprocessorPlan(sft: SimpleFeatureType,
   override def scan(ds: HBaseDataStore): CloseableIterator[SimpleFeature] = {
     import org.locationtech.geomesa.index.conf.QueryHints.RichHints
     if (hints.isDensityQuery) {
-      val is: Map[String, String] = KryoLazyDensityCoprocessor.configure(sft, ranges, hbaseFilters, hints)
+      val rowRanges = Lists.newArrayList[RowRange]()
```

Comment:
I think it would be more consistent to move this out of the scan plan class and into HBasePlatform class - as I noted before, remote filters are configured there for most scan plans, so including it in the ScanPlan is confusing